frontend/packages/shared/src/components/DragAndDropTree/DragAndDropTreeItem/DragAndDropTreeItem.tsx

```diff
@@ -24,7 +29,7 @@ export const DragAndDropTreeItem = ({
   const { hoveredNodeParent, setHoveredNodeParent } = useContext(DragAndDropTreeRootContext);
   const { nodeId: parentId } = useContext(DragAndDropTreeItemContext);
 
-  const subList = children ? <DragAndDrop.List>{children}</DragAndDrop.List> : null;
+  const isExpandable = expandable || !!children;
```

Comment:
Dette er småpirk, men er dette mer leselig? 😊 

```suggestion
  const isExpandable = expandable || Boolean(children);
```